parcels/examples/tutorial_SummedFields.ipynb

```diff
@@ -234,7 +234,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.9.5"
   }
```

Comment:
hm - is it possible to revert that ? just asking, due to backward compatibility.

Reply:
Done, my mistake.